index.js

```diff
@@ -31,6 +33,41 @@ async function run() {
     }
     containerDef.image = imageURI;
 
+    // Insert environment variable, if present
+    if (environmentVariables) {
+
+      // If environment array is missing, create it
+      if (!Array.isArray(containerDef.environment)) {
+        containerDef.environment = [];
+      }
+
+      // Get pairs by splitting with |
+      environmentVariables.split('|').forEach(function (item) {
+        // Split pairs on =
+        const pair = item.split('=');
+        // If that pair didn't have exactly 2 parts (name and value)
+        if (pair.length != 2) {
+          throw new Error('Each environment-variable pair must be of form NAME=value');
```

Comment:
should we add the problematic `item` here so that it's clear to the user why we couldn't split?
```suggestion
          throw new Error(`Cannot parse the environment variable '${item}'. Environment variable pairs must be of the form NAME=value.`);
```